deploy/manifests/custom-metrics-apiserver-deployment.yaml

```diff
@@ -40,6 +43,12 @@ spec:
           readOnly: true
         - mountPath: /tmp
           name: tmp-vol
+        - mountPath: /etc/tls
+          name: metricstlsconfig
+          readOnly: false
+        - mountPath: /etc/certs
+          name: servingcerts
+          readOnly: false
```

Comment:
we shouldn't need these certs

Reply:
By default, I don't think we should add certs since ultimately they will have to be managed by the users so they will have to add their own TLS config if they want to setup the server properly. In the meantime we can have the metrics exposed over HTTPS but we can skip the cert verification by setting `insecureSkipTLSVerify: true`.

Reply:
Done

Reply:
The insecureSkipTLSVerify: true is already present at the ApiService level.